File: Pallindrome.c
// Check it is pallindrome or not

#include<stdio.h>

int main()
{
    int iNo = 0;
    printf("Enter number\n");
    scanf("%d",&iNo);
    int iDigit = 0;
    int iRev = 0;
    int iTemp = iNo;

    

    while(iNo != 0)
    {
        iDigit = iNo % 10;
        iRev = (iRev*10)+iDigit;
        iNo = iNo/10;
    }
    if(iRev == iTemp)
    {
        printf("Pallindrome no\n");
    }
    else
    {
        printf("Not Pallindrome\n");
    }

    return 0;
}

File: Pallindrome.java
import java.util.*;

public class Pallindrome 
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enterr number");
        int iNo = sobj.nextInt();

        Number nobj = new Number();

        boolean bret = nobj.Pallindrome(iNo);
        if(bret==true)
        {
            System.out.println("It is Pallindrome");
        }
        else
        {
            System.out.println("It is not pallindrome");
        }

    }
}

class Number
{
    public boolean Pallindrome(int iValue)
    {
        int iTemp = iValue;

        int iDigit=0;
        int iRev=0;

        while(iValue > 0)
        {
            iDigit = iValue%10;
            iRev = (iRev*10)+iDigit;
            iValue = iValue/10;
        }

        if(iTemp==iRev)
        {
            return true;
        }
        else
        {
            return false;
        }

    }
}